pkg/api/latest/dynakube/extensions/spec.go

```diff
@@ -6,23 +6,26 @@ import (
 )
 
 type Extensions struct {
-	*ExecutionControllerSpec
+	Controller *ExecutionControllerSpec
 
 	name      string
 	namespace string
-	enabled   bool
+
+	Databases []DatabaseSpec
+
+	enabled bool
 }
 
 // +kubebuilder:object:generate=true
 type Spec struct {
-	PrometheusSpec *PrometheusSpec `json:"prometheus,omitempty"`
+	Prometheus *PrometheusSpec `json:"prometheus,omitempty"`
```

Comment:
any reason why you don't line `*Spec` suffix?

Reply:
it's redundant info and is inconsistent with the rendered JSON